src/services/databaseServices/createFacebookUser.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from "../../models/user";
+import { logger } from "../errorLogger";
+
+export const createFacebookUser = async ({
+  username,
+  password,
+  email,
+}: Partial<User>) => {
+  try {
+    return await User.create({
+      username,
+      password,
+      email,
+    });
+  } catch (error) {
```

Comment:
Why do you swallow errors? If `create` fails this will usually be because of unique constraint violations or connection errors - I think it's a valid reason to throw and show return a 500 to the endpoint consumer.

Otherwise the endpoint may return a success response even though the user was not created (yikes)